src/guide/modules.md

```diff
@@ -100,32 +100,32 @@ from "list" include List as Stdlist
 Stdlist.length([1, 2, 3]) // 3
 ```
 
-### Including All Values
+### Bring Values from a Module into Scope
 
-If we want to bring all of the values from another module into scope in our program, we can use an asterisk.
+Individual items in a module can be brought into scope by using the `use` keyword, placing the desired items in curly braces. Similar to with the `provide { ... }` syntax, we need to prefix types with `type` and modules with `module`
 
 ```grain
 module Main
 
-from "list" include List
+from "array" include Array
 
-use List.*
+use Array.{ length, module Immutable }
 
-length([1, 2, 3]) // 3
-reverse([1, 2, 3]) // [3, 2, 1]
+length([> 1, 2, 3])
+let imm = Immutable.fromList([1, 2, 3])
```

Comment:
I think it would be good if we showed the syntax used on submodules too like:
```
use Immutable.{ length }
assert length(imm) == 3
```

What might be even better is a final section at the end where we are like this all works on submodules too and then show all the syntax being used on a submodule.